analytics_data_api/v0/exceptions.py

```diff
@@ -0,0 +1,52 @@
+import abc
+
+
+class BaseError(Exception):
+    """
+    Base error.
+    """
+
+    __metaclass__ = abc.ABCMeta
```

Comment:
Why do we need `abc`? I thought that the `Exception` class provides this behavior on its own.